### DISCUSSION TITLE
configure renovate pull request boundaries

### DISCUSSION THREAD

**Discussion by alaamub** [renovatebot/renovate#11512] [Request Help]
Hi, is it possible to configure renovate so the generated pull request can ignore existing comments in requirements.txt and not delete them?

Example:
```
- requests==2.25.1 \
-     --hash=sha256:27973dd4a904a4f13b263a19c866c13b92a39ed1c964655f025f3f8d3d75b804 \
-     --hash=sha256:c210084e36a42ae6b9219e00e48287def368a26d03a048ddad7bfee44f75871e
-     # via -r myproject/requirements.in
+ requests==2.26.0 \
+   --hash=sha256:6c1246513ecd5ecd4528a0906f910e8f0f9c6b8ec72030dc9fd154dc1a6efd24 \
+    --hash=sha256:b8aa58f8cf793ffd8782d3d8cb19e66ef36f7aba4353eec859e74678b01b07a7
```
As you can see above, renovate removed the comment `# via -r myproject/requirements.in`.

**Comment by rarkins** [comment]
Not configurable but should be fixable

**Comment by alaamub** [comment]
Thanks @rarkins. should I wait for this to be fixed in the newer version or should I issue a PR?

**Reply by viceice** [reply]
Please create a bug issue first and please provide a minimal reproduction.

**Reply by alaamub** [reply]
Thanks, created one https://github.com/renovatebot/renovate/issues/11552
